### DISCUSSION TITLE
Allow osu!lazer to update automatically on macOS

### DISCUSSION THREAD

**Discussion by SchiavoAnto** [ppy/osu#25879] [Ideas]
On macOS, when a new update is released, osu!lazer does not update automatically like on Windows, but it requires the user to download the new update zip file and replace the .app in the Applications folder manually.
I feel like this is a bad experience, especially for new players, because it adds inconsistency between platforms.

**Comment by peppy** [comment]
Needs an upgrade to the new version of squirrel.
